packages/components/src/tooltip/stories/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import { text, select } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import Tooltip from '../';
+
+export default {
+	title: 'Components/ToolTip',
+	component: Tooltip,
+};
+
+export const _default = () => {
```

Comment:
Maybe we can just remove this one and just keep the "withPosition" one as default?

Reply:
No problem, updated